src/app/debug/table/table.component.ts

```diff
@@ -74,6 +74,9 @@ export class TableComponent implements OnInit, OnDestroy {
   defaultCheckBoxSize: number = 13;
   defaultFontSize: number = 8;
   fontSizeSpacingModifier: number = 1.2;
+  hasTimedOut: boolean = false;
+  reportsInProgress: { [key: string]: number } = {};
```

Comment:
You can use the `Record<string, number>` type for this, its not mandatory but might be more readable and consistent with other projects that do use that type